ramls/override-renewal-by-barcode-request.json

```diff
@@ -19,6 +19,11 @@
     "comment": {
       "description": "Comment to override action stored in loan history",
       "type": "string"
+    },
+    "overrideServicePointId": {
+      "description": "Service point id where override happens",
+      "type": "string",
```

Comment:
Can it be a UUID rather than a string?

Reply:
we don't have a UUID type in schema, however there is `"$ref": "raml-util/schemas/uuid.schema"` right below, which applies some validation.

Reply:
Apologies, that is what I mean by a type. I think of those as user defined types.